python/paddle/tensor/linalg.py

```diff
@@ -3745,3 +3745,133 @@ def cdist(
     return paddle.linalg.norm(
         x[..., None, :] - y[..., None, :, :], p=p, axis=-1
     )
+
+
+def householder_product(x, tau, name=None):
+    r"""
+
+    Computes the first n columns of a product of Householder matrices.
+
+    This function can get the vector :math:`\omega_{i}` from matrix `x`(m x n), the :math:`i-1` elements are zeros, and the i-th is `1`, the rest of the elements are from i-th column of `x`.
```

Comment:
```suggestion
    This function can get the vector :math:`\omega_{i}` from matrix `x` (m x n), the :math:`i-1` elements are zeros, and the i-th is `1`, the rest of the elements are from i-th column of `x`.
```
预览有点问题
![image](https://github.com/PaddlePaddle/Paddle/assets/70642955/150505ac-198b-418b-8a23-daa3f95f9f51)

Reply:
诶想问下英文文档渲染结果一般怎么看呢？

Reply:
> 诶想问下英文文档渲染结果一般怎么看呢？

1. doc-preview CI跑完后， 点进 `details`的日志里
![image](https://github.com/PaddlePaddle/Paddle/assets/70642955/a00a0ebb-dd4a-40a6-86d6-55e5b47c2440)

2. 搜 `preview` 关键词
![image](https://github.com/PaddlePaddle/Paddle/assets/70642955/d012d3af-dcf2-4bc0-b897-a2a55baab880)

-----
或者直接用PR号替换url  `http://preview-paddle-pr-<PR号>.paddle-docs-preview.paddlepaddle.org.cn/documentation/docs/en/api/paddle/abs_en.html`

如本PR号为 `58214`。则渲染链接为 `http://preview-paddle-pr-58214.paddle-docs-preview.paddlepaddle.org.cn/documentation/docs/en/api/paddle/abs_en.html`

Reply:
有本地渲染方案嘛？每次要等ci的话感觉有点麻烦

Reply:
> 有本地渲染方案嘛？每次要等ci的话感觉有点麻烦

暂时没有，或者你要不来帮我们重构一下文档，就能实现了（狗头

替代方案就是可以vscode下个rst 预览的插件，docstring 都是rst格式的，遵循sphinx语法，rst 预览没太大问题应该就问题不大

Reply:
> > 有本地渲染方案嘛？每次要等ci的话感觉有点麻烦
> 
> 暂时没有，或者你要不来帮我们重构一下文档，就能实现了（狗头
> 
> 替代方案就是可以vscode下个rst 预览的插件，docstring 都是rst格式的，遵循sphinx语法，rst 预览没太大问题应该就问题不大

哈人    那我先vscode插件苟一下